adapters/beachfront/beachfront.go

```diff
@@ -394,12 +401,33 @@ func getVideoRequest(req *openrtb.BidRequest) (BeachfrontVideoRequest, []error,
 func (a *BeachfrontAdapter) MakeBids(internalRequest *openrtb.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
 	var bids []openrtb.Bid
 	var bidtype = getBidType(internalRequest)
+
+	/*
+		Beachfront is now sending an empty array and 200 as their "no results" response. This should catch that.
+	*/
+
+	if response.StatusCode == http.StatusOK && len(response.Body) <= 2 {
```

Comment:
Is there a reason why we can't just check `len(response.Body) == 0` rather than `len(response.Body) <= 2` ?